python/cudf/cudf/utils/utils.py

```diff
@@ -58,7 +58,7 @@ def check_equals_int(a, b):
 
 
 def scalar_broadcast_to(scalar, size, dtype=None):
-    from cudf.utils.dtypes import to_cudf_compatible_scalar, is_string_dtype
+    from cudf.utils.dtypes import to_cudf_compatible_scalar
     from cudf.core.column import column_empty
```

Comment:
Move these imports out of the function or just use fully qualified name within the function.